### ISSUE TITLE
Target time-mean maps

### ISSUE THREAD

**Issue by gideonite** [ai2cm/ace#215]
Currently we only output gen time-mean maps but this makes it difficult to sanity check the outputs. We need to include target time-mean maps to help with a stable comparison.

**Comment by oliverwm1** [comment]
We also output the time-mean bias map, so looking at this in conjunction with the gen time-mean map gives same information as target (although I agree sometimes useful to compare to non-bias maps).

**Comment by gideonite** [comment]
Agreed. @mcgibbon and I were looking at this earlier today and having some trouble sanity checking some results though, which is what motivated opening this issue.

**Comment by oliverwm1** [comment]
We have been operating fine without these maps for a couple years, so going to close this issue for now.
